pages/positions/consulting-software-engineer.md

```diff
@@ -27,8 +27,14 @@ supervisory_status: 'No'
 ---
 
 {% if page.state == 'upcoming' %}
-{{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
-{% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?topic_id=USGSATTS_6). {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process. 
+  {{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
+  {% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://goo.gl/forms/QMbyPse8f4rfnZ9z2).
```

Comment:
Notice the whitespace in front of the 2nd if statement -- i think that is requried, and might be why you were having problems w/ nested if statements.

Reply:
Ah, yeah that could have been it.